packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -56,6 +73,9 @@ TimestampContainer.propTypes = {
   script: shape(scriptPropType).isRequired,
   locale: string,
   service: string.isRequired,
+  altCalendar: shape({
+    formatDate: func,
```

Comment:
Could this be `func.isRequired` to be stricter?

Also we should add documentation for the new prop.

Reply:
I originally had this, but then I thought if a new calendar is added, something other than `formatDate` could be used as a function for formatting. 

Which made me think that we might have to refactor `psammead-calendars` so that all future calendars will make use of a `formatDate` function so we can use `func.isRequired`.

Reply:
At the moment the only way an alt calendar can be used is by calling `formatDate` like on line https://github.com/bbc/psammead/pull/2303/files#diff-9167111d8797dcd3c0e7444464a1d1c7R44 so I think it's fine to add `isRequired` for some extra help debugging. If we expand the calendar API later it's easy to change.

Reply:
Okay I'll change it to required